spec/features/create_new_work_spec.rb

```diff
@@ -35,7 +33,7 @@
       expect(page).to have_content 'Deposit your content'
 
       # Test client-side validation messages
-      fill_in 'Publication year', with: '2021'
+      fill_in 'Publication year', with: '2049'
```

Comment:
because otherwise i think the test would stop working when we get back from break?

Reply:
:laughing: good call